docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/creating-vehicle-description.md

```diff
@@ -0,0 +1,205 @@
+    wheel_base: 2.79 # between front wheel center and rear wheel center
+    wheel_tread: 1.64 # between left wheel center and right wheel center
+    front_overhang: 1.0 # between front wheel center and vehicle front
+    rear_overhang: 1.1 # between rear wheel center and vehicle rear
+    left_overhang: 0.128 # between left wheel center and vehicle left
+    right_overhang: 0.128 # between right wheel center and vehicle right
+    vehicle_height: 2.5
+    max_steer_angle: 0.70 # [rad]
+```
+
+The tutorial_vehicle file is [vehicle_info.param.yaml](https://github.com/leo-drive/tutorial_vehicle_launch/blob/main/tutorial_vehicle_description/config/vehicle_info.param.yaml)
+included in [tutorial_vehicle_launch](https://github.com/leo-drive/tutorial_vehicle_launch/tree/main) and given below:
+
+??? note "vehicle_info.param.yaml for tutorial_vehicle_launch"
+
+    ```yaml
+    /**:
+      ros__parameters:
+        wheel_radius: 0.27855 # The radius of the wheel, primarily used for dead reckoning.
+        wheel_width: 0.155 # The lateral width of a wheel tire, primarily used for dead reckoning.
+        wheel_base: 1.60 # between front wheel center and rear wheel center
+        wheel_tread: 0.959 # between left wheel center and right wheel center
+        front_overhang: 0.245 # between front wheel center and vehicle front
+        rear_overhang: 0.245 # between rear wheel center and vehicle rear
+        left_overhang: 0.075 # between left wheel center and vehicle left
+        right_overhang: 0.075 # between right wheel center and vehicle right
+        vehicle_height: 2.
+        max_steer_angle: 0.70 # [rad]
+    ```
+
+### 4. vehicle.fbx
+
+`vehicle.fbx` is a 3D model file used for visualization in RVIZ.
+You must add the fbx or dae file of your own vehicle.
+
+The 3D model of Lexus vehicle prepared for sample_vehicle_launch is [here](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_description/mesh).
+
+### 5. vehicle.xacro
+
+`vehicle.xacro` is a macro file that defines the vehicle model.
+
+The original [vehicle.xacro](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_description/urdf/vehicle.xacro) file for sample_vehicle_launch is given below:
+
+??? note "vehicle.xacro for sample_vehicle_launch"
+
+    ```yaml
+    <?xml version="1.0"?>
+    <robot xmlns:xacro="http://ros.org/wiki/xacro">
+      <!-- load parameter -->
+      <xacro:property name="vehicle_info" value="${xacro.load_yaml('$(find sample_vehicle_description)/config/vehicle_info.param.yaml')}"/>
+
+      <!-- vehicle body -->
+      <link name="base_link">
+        <visual>
+          <origin xyz="${vehicle_info['/**']['ros__parameters']['wheel_base']/2.0} 0 0" rpy="${pi/2.0} 0 ${pi}"/>
+          <geometry>
+            <mesh filename="package://sample_vehicle_description/mesh/lexus.dae" scale="1 1 1"/>
+          </geometry>
+        </visual>
+      </link>
+    </robot>
+    ```
+
```

Comment:
This section need to be more detailed. Where is our tutorial_vehicle .fbx model? Where is the directory which I put .fbx file on? What I need to change vehicle.xacro for my own vehicle? This part need to be more detailed.